include/seqan3/io/sam_file/detail/format_sam_base.hpp

```diff
@@ -481,9 +480,9 @@ inline void format_sam_base::read_header(stream_view_type && stream_view,
                         else
                             throw format_error{std::string{"Illegal SAM header tag: S"} + *it};
```

Comment:
We should not throw for unknown tags starting with S (e.g. SL or whatever) but just ignore them.